tools/rna_tools/dot2ct/datatypes_conf.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<datatypes>
```

Comment:
@yhoogstrate can we stick to 4spaces intent for all files? This is for sure debatable, but it is kind of a standard.